src/block/block_addr.c

```diff
@@ -354,28 +354,28 @@ __wt_ckpt_verbose(WT_SESSION_IMPL *session, WT_BLOCK *block, const char *tag, co
         WT_ERR(__wt_buf_catfmt(session, tmp, ", root=[Empty]"));
     else
         WT_ERR(__wt_buf_catfmt(session, tmp,
-          ", root=[%" PRIuMAX "-%" PRIuMAX ", %" PRIu32 ", %" PRIu32 "]",
+          ", root=[off: %" PRIuMAX "-%" PRIuMAX ", size: %" PRIu32 ", checksum: 0x%08x]",
```

Comment:
Please use `PRIx32` and/or friends here and in other places.

Reply:
hi, @ershov 
Thank you for your guidance，I have change the code, please review again.